src/main/webapp/i18n/de/exam.json

```diff
@@ -62,37 +62,37 @@
                 "atLeastOneExercisePerExerciseGroup": "Alle Aufgabengruppen müssen mindestens eine Aufgabe haben.",
                 "allExercisesInExerciseGroupOfSameIncludedType": "Alle Aufgaben in einer Aufgabengruppe müssen alle die selbe Bedeutung haben für das Klausurergebnis (Normale Aufgabe, Bonusaufgabe, Keine Bedeutung).",
                 "allExercisesInExerciseGroupGiveSameNumberOfPoints": "Alle Aufgaben in einer Aufgabengruppe müssen dieselbe Anzahl an Punkte oder Bonuspunkte geben",
-                "tooManyMaxPoints": "Überprüfe die maximale Punkteanzahl in der Klausur! Die maximalen Punkte die ein Student in den verpflichtenden Aufgaben erreichen kann ist zu gross.",
-                "tooFewMaxPoints": "Überprüfe die maximale Punkteanzahl in der Klausur! Die maximalen Punkte die ein Student in den Aufgaben erreichen kann ist zu niedrig.",
+                "tooManyMaxPoints": "Überprüfe die maximale Punktzahl in der Klausur! Die maximale Punktzahl, die Studierende in den verpflichtenden Aufgaben erreichen können, ist zu groß.",
+                "tooFewMaxPoints": "Überprüfe die maximale Punktzahl in der Klausur! Die maximale Punktzahl, die Studierende in den Aufgaben erreichen können, ist zu niedrig.",
                 "numberOfExercisesMustBeSet": "Die Anzahl der Aufgaben für die Klausur muss gesetzt sein.",
                 "numberOfExercisesInExamNotSet": "Die Anzahl an Aufgaben in der Klausur ist nicht gesetzt.",
                 "tooFewExerciseGroups": "Die Anzahl der Aufgabengruppen ist zu klein.",
                 "tooManyMandatoryExerciseGroups": "Die Anzahl verpflichtender Aufgabengruppen ist zu groß.",
                 "maxPointsNotSet": "Die maximale Punktzahl in der Klausur ist nicht gesetzt."
             },
             "error": {
-                "cannotRegisterInstructor": "Du kannst keine Dozenten oder Administratoren zu Klausuren anmelden."
+                "cannotRegisterInstructor": "Du kannst keine Lehr- oder Administrationspersonen zu Klausuren anmelden."
```

Comment:
I don't think Administrationsperson is a German word :)

s.o. Lehrkraft

Reply:
found an alternative in  51600a0